app/styles/constants.css

```diff
@@ -19,3 +19,9 @@
     --novnc-green: rgb(0, 128, 0);
     --novnc-yellow: rgb(255, 255, 0);
 }
+
+/* ---------- SIZES ----------- */
+
+:root {
+    --border-width: 1px;
```

Comment:
This also seems superfluous to have a variable for. It's not like we're going to fiddle around with this?